[2021-03-09 17:38:53,939] [log.py|console] [line:54] ERROR   : 40
[2021-03-09 17:42:40,949] [log.py|console] [line:54] ERROR   : 40
[2021-03-09 17:43:09,898] [log.py|console] [line:54] ERROR   : sdfsdfsdfsdfsdfsdffsfsdfsdf
